mantidproject/mantid
# remove deprecated and disabled algorithms

fixes #9020 
Removed the Old and disabled versions of LoadRaw and LoadISISNexus
## Release Notes

Too minor
## To Test
1. Ensure unit and system tests pass
2.  Start up Mantidplot, check there is only version 3 of LoadRaw, and Version 2 of LoadISIS Nexus
